samples/grids/data-grid/accessibility/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-data-grid-binding-local-data",
+  "description": "This project provides example of Data Grid Binding Local Data using Infragistics React components",
+  "author": "Infragistics",
+  "homepage": ".",
+  "version": "1.4.0",
+  "private": true,
+  "scripts": {
+    "start": "set PORT=4200 && react-scripts --max_old_space_size=10240 start",
+    "build": "react-scripts --max_old_space_size=10240 build ",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "dependencies": {
+    "@infragistics/igniteui-react-core": "20.2.145",
+    "@infragistics/igniteui-react-grids": "20.2.145",
+    "@infragistics/igniteui-react-inputs": "20.2.145",
+    "@infragistics/igniteui-react-layouts": "20.2.145",
```

Comment:
you need to target these dependencies:

    "igniteui-react-core": "16.12.4",
    "igniteui-react-grids": "16.12.4",
    "igniteui-react-inputs": "16.12.4",
    "igniteui-react-layouts": "16.12.4",

Reply:
added to latest commits